### PR TITLE
Fix yarn install, lock redstone-node, update other deps

### PR DESCRIPTION
**Pull request by goriunov**
This PR closes https://github.com/metaplex-foundation/metaplex/issues/1672 exact issues is explained in the ticket itself.

### CODE REVIEW

**Comment by kiankyars** [comment]
I am having this issue on a m1 MacBook Air where yarn install gives the following error:
Screen Shot 2022-01-30 at 1 09 13 PM

**Review by stegaBOB — APPROVED** [review]

**Comment by Deliceli58** [comment]
and how can I lock redstone-node, update other deps? which queue of commands I need to use

**Comment by GenMusic** [comment]
> 

Yeah, still getting the same issue.
